src/amo/components/AddonMoreInfo/index.js

```diff
@@ -378,6 +444,7 @@ const AddonMoreInfo: React.ComponentType<Props> = compose(
   withRouter,
   translate(),
   connect(mapStateToProps),
+  withErrorHandler({ name: 'AddonMoreInfo' }),
```

Comment:
When adding a new error handler, we should also render errors in the UI:

```diff
diff --git a/src/amo/components/AddonMoreInfo/index.js b/src/amo/components/AddonMoreInfo/index.js
index 22830e7a9..aba6cd9cf 100644
--- a/src/amo/components/AddonMoreInfo/index.js
+++ b/src/amo/components/AddonMoreInfo/index.js
@@ -385,10 +385,12 @@ export class AddonMoreInfoBase extends React.Component<InternalProps> {
   }

   render(): React.Node {
-    const { i18n } = this.props;
+    const { errorHandler, i18n } = this.props;

     return (
       <Card className="AddonMoreInfo" header={i18n.gettext('More information')}>
+        {errorHandler.renderErrorIfPresent()}
+
         {this.listContent()}
       </Card>
     );
```

Reply:
And we need to cover these changes with tests.

Reply:
I was looking at other tests that had `renderErrorIfPresent()`, but only found it in `tests/unit/amo/test_errorHandler.js`. .. would the test added in the latest push (`renders errors when API error occurs`) cover this?